frontend/src/pages/chat/chat.tsx

```diff
@@ -0,0 +1,87 @@
+import { FormEvent, useEffect, useState } from "react";
+import { RoomsList } from "../../components/rooms-list";
+import { Room } from "../../types/room";
+import "./chat.scss";
+import { Avatar } from "../../components/avatar";
+import { useParams, useNavigate } from "react-router-dom";
+import { useUser } from "../../hooks/useUser";
+import { MessageList } from "../../components/message-list";
+import { useData } from "../../data/useData";
+import axios from "axios";
+import { NewRoomForm } from "../../components/new-room-form";
+import { CurrentRoom } from "../../components/current-room";
+
+export const ChatPage = () => {
+  const { chatName } = useParams();
+  const navigate = useNavigate();
+
+  const {
+    data: { rooms },
+  } = useData();
+
+  const { userName, deleteUserName } = useUser();
+  const [newMessage, setNewMessage] = useState("");
+
+  const color = "black";
+  const currentRoom = rooms.find((room) => room.name === chatName);
+
+  const handleSelectRoom = (room: Room) => {
+    navigate(`/chat/${room.name}`);
+  };
+
+  const handleSubmitNewMessage = (e: FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    axios.post("http://localhost:4000/messages", {
```

Comment:
The result of the axios post request is not handled. It's good practice to handle the promise that is returned by axios functions. You can use .then() to handle the response or .catch() to handle any errors.